tests/config/ray-cluster.mini.yaml.template

```diff
@@ -12,38 +12,18 @@ spec:
   headGroupSpec:
     # Kubernetes Service Type, valid values are 'ClusterIP', 'NodePort' and 'LoadBalancer'
     serviceType: ClusterIP
-    # the pod replicas in this group typed head (assuming there could be more than 1 in the future)
-    replicas: 1
-    # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
+    # the following params are used to complete the ray start: ray start --head --block ...
     rayStartParams:
-      port: '6379' # should match headService targetPort
-      #include_webui: 'true'
-      redis-password: 'LetMeInRay' # Deprecated since Ray 1.11 due to GCS bootstrapping enabled
-      # webui_host: "10.1.2.60"
       dashboard-host: '0.0.0.0'
-      num-cpus: '1' # can be auto-completed from the limits
-      node-ip-address: $$MY_POD_IP # auto-completed as the head pod IP
+      num-cpus: '1'
       block: 'true'
     #pod template
     template:
       metadata:
-        labels:
-          # custom labels. NOTE: do not define custom labels start with `raycluster.`, they may be used in controller.
-          # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
-          rayCluster: raycluster-compatibility-test
```

Comment:
`rayCluster: raycluster-compatibility-test` cannot be removed in the head pod. It is used in the NodePort service.

https://github.com/ray-project/kuberay/blob/3a647129e248640f2314bfafd257f6b7a5db232f/tests/config/raycluster-service.yaml#L8

Reply:
I fixed it to use the selector used by the built-in service
https://github.com/ray-project/kuberay/pull/761/commits/f6b43d7cbd46636c61f5e0c19a46408049cfe90e

I tried to remove the extra service by using spec.headGroupSpec.ServiceType but that didn't work out.

Reply:
Actually, switched it back to how it was. Changed the label to something more descriptive and added explanation of what it's for.